Replace "simple" emojis shortcodes (#372)
ServiceNow/azimuth: docs/docs/user-guide/index.md

* The **project name** from the config file is shown.
* A dropdown :material-arrow-down-drop-circle-outline: allows you to select the different pipelines
  defined in the config. It also allows you to select no pipelines.

> * The [**settings**](settings.md) :fontawesome-solid-gear: allow you to enable/disable different analyses.

Reviewer: Will that one work? Because it's not an emoji?

Author: Correct, all the _fontawesome_, _material_ and _octicons_ icons kept working.

> <!-- TODO The settings do more than that now. -->

Reviewer: I'm afraid we'll forget to remove this comment before the next release. Can we already update it here?

> * The [**settings**](settings.md) :fontawesome-solid-gear: allow you to enable/disable different analyses.
> <!-- TODO The settings do more than that now. -->

Reviewer: ```suggestion
* The [**settings**](settings.md) :fontawesome-solid-gear: allow the user to edit the config file.
```

Author: Oh if that's all, of course! 😝 Thank you!
I will
* conjugate "allow" with an "s";
* keep "you" instead of "the user" to match the other bullets; and
* remove the word "file" to avoid confusion since we are not altering the file on disc, just the config loaded in memory. 

```suggestion
* The [**settings**](settings.md) :fontawesome-solid-gear: allows you to edit the config.
```

Reviewer: I think in this location that's it yes. We'll need to edit the settings documentation before releasing though! :D
